aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -4,156 +4,68 @@ author: rick-anderson
-
-![API Manager Credentials page: Download credentials](index/_static/GoogleConsoleFinish.png)
-
-* Tap **Download** to save a JSON file with application secrets, and **Done** to complete creation of the new app.
-
-* When deploying the site you'll need to revisit the **Google Console** and register a new public url.
+* Navigate to [Integrating Google Sign-In into your web app](https://developers.google.com/identity/sign-in/web/devconsole-project) and select **CONFIGURE A PROJECT**.
+* In the **Configure your OAuth client** dialog, select **Web server**.
+* In the **Authorized redirect URIs** text entry box, set the redirect URI. For example, `https://localhost:5001/signin-google`
+* Save the **Client ID** and **Client Secret**.
+* When deploying the site, register the new public url from the **Google Console**.
 
 ## Store Google ClientID and ClientSecret
 
-Link sensitive settings like Google `Client ID` and `Client Secret` to your application configuration using the [Secret Manager](xref:security/app-secrets). For the purposes of this tutorial, name the tokens `Authentication:Google:ClientId` and `Authentication:Google:ClientSecret`.
-
-The values for these tokens can be found in the JSON file downloaded in the previous step under `web.client_id` and `web.client_secret`.
-
-## Configure Google Authentication
-
-::: moniker range=">= aspnetcore-2.0"
-
-Add the Google service in the `ConfigureServices` method in *Startup.cs* file:
-
-```csharp
-services.AddDefaultIdentity<IdentityUser>()
-        .AddDefaultUI(UIFramework.Bootstrap4)
-        .AddEntityFrameworkStores<ApplicationDbContext>();
+Store sensitive settings such as the Google `Client ID` and `Client Secret` with the [Secret Manager](xref:security/app-secrets). For the purposes of this tutorial, name the tokens `Authentication:Google:ClientId` and `Authentication:Google:ClientSecret`:
 
-services.AddAuthentication().AddGoogle(googleOptions =>
-{
-    googleOptions.ClientId = Configuration["Authentication:Google:ClientId"];
-    googleOptions.ClientSecret = Configuration["Authentication:Google:ClientSecret"];
-});
+```console
+dotnet user-secrets set "Authentication:Google:ClientId" "X.apps.googleusercontent.com"
+dotnet user-secrets set "Authentication:Google:ClientSecret" "<client secret>"
 ```
 
-[!INCLUDE [default settings configuration](includes/default-settings.md)]
+You can manage your API credentials and usage in the [API Console](https://console.developers.google.com/apis/dashboard).
 
-[!INCLUDE[](includes/chain-auth-providers.md)]
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-2.0"
+## Configure Google authentication
 
-The project template used in this tutorial ensures that [Microsoft.AspNetCore.Authentication.Google](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Google) package is installed.
+Add the Google service to `Startup.ConfigureServices`. See the code in [this GitHub issue](https://github.com/aspnet/AspNetCore/issues/6486).
```

Comment:
```suggestion
Add the Google service to `Startup.ConfigureServices`.
```
The link is redundant, you already have it at the top. At some point after everyone has updated we'll remove that top paragraph and be done with it.